api/tests/python/tests/formats/molecules/cis_transi.ket

```diff
@@ -0,0 +1 @@
+{"root":{"nodes":[{"$ref":"mol0"}]},"mol0":{"type":"molecule","atoms":[{"label":"F","location":[1.5260000228881837,-1.5499999523162842,0.0]},{"label":"C","location":[2.3919999599456789,-2.049999952316284,0.0]},{"label":"C","location":[3.257999897003174,-1.5499999523162842,0.0]},{"label":"F","location":[4.124000072479248,-2.049999952316284,0.0]}],"bonds":[{"type":1,"atoms":[0,1]},{"type":2,"atoms":[1,2],"stereo":8},{"type":1,"atoms":[2,3]}]}}
```

Comment:
This file looks like a version of previous files `cis_trans.ket` or `cis_trans.ket.json` with some rounds, new field `stereo` and compressed. Do we need it?